src/theme.module.css

```diff
@@ -42,12 +61,72 @@
 	--brown: themeBrown;
 	--grey: themeGrey;
 
-	--theme-text: rgba(0, 0, 0, 0.87); /* Matching semantic for now */
-	--theme-background: themeWhite;
-	--theme-backdrop: themeLight;
 	--theme-inverse: themeBlack;
 }
-@value themeText: var(--theme-text);
-@value themeBackground: var(--theme-background);
-@value themeBackdrop: var(--theme-backdrop);
-@value themeInverse: var(--theme-inverse);
+
+.light-mode {
+	/* Basic Light Mode values */
+	--text: rgba(0, 0, 0, 0.87);
+	--background: themeWhite;
+	--backdrop: themeLight;
+	--text-muted: #666;
+	--segment-active: #e1e4e6;
+	--border-color: #dbdbdb;
+	/* Light Mode Message Box values */
+	--message-background: messageLight;
+	--message-text: messageDark;
+	--info-background: infoLight;
+	--info-text: infoDark;
+	--success-background: successLight;
+	--success-text: successDark;
+	--warning-background: warningLight;
+	--warning-text: warningDark;
+	--error-background: errorLight;
+	--error-text: errorDark;
+
+	color: var(--text);
+	background: var(--backdrop);
+}
+
+.dark-mode {
+	/* Basic Dark Mode values */
+	--text: rgba(255, 255, 255, 0.87);
+	--background: #2d2d2d;
+	--backdrop: #3d3d3d;
+	--text-muted: #aaa;
+	--segment-link-active: #202020;
+	--border-color: #7e7e7e;
+	/* Dark Mode Message Box values */
+	--message-background: messageDark;
+	--message-text: messageLight;
+	--info-background: infoDark;
+	--info-text: infoLight;
+	--success-background: successDark;
+	--success-text: successLight;
+	--warning-background: warningDark;
+	--warning-text: warningLight;
+	--error-background: errorDark;
+	--error-text: errorLight;
+
+	color: var(--text);
+	background: var(--backdrop);
+}
+
+
+@value themeText: var(--text);
```

Comment:
`themeText` and `themeBackdrop` are showing up as new here. I changed the CSS var to be a bit less wordy.